Lib/sqlite3/test/dbapi.py

```diff
@@ -85,7 +85,12 @@ def test_not_supported_error(self):
     def test_shared_cache_deprecated(self):
         for enable in (True, False):
             with self.assertWarns(DeprecationWarning) as cm:
-                sqlite.enable_shared_cache(enable)
+                try:
+                    sqlite.enable_shared_cache(enable)
+                except sqlite.OperationalError as err:
+                    # See issue 24464 and https://sqlite.org/c3ref/enable_shared_cache.html
+                    import sys
+                    self.assertEqual(sys.platform, 'darwin')
```

Comment:
I'd rather skip the test here if OperationalError was caused by:

> This method is disabled on MacOS X 10.7 and iOS version 5.0 and will always return SQLITE_MISUSE.

Reply:
Also, third-party links tend to change so I'd prefer explaining why we are doing this in a comment instead of just linking to SQLite docs.